tasks/notify.py

```diff
@@ -48,12 +50,15 @@ def check_teams(_):
 
 
 @task
-def send_message(_, notification_type="merge", print_to_stdout=False):
+def send_message(ctx, notification_type="merge", print_to_stdout=False):
     """
     Send notifications for the current pipeline. CI-only task.
     Use the --print-to-stdout option to test this locally, without sending
     real slack messages.
     """
+    default_branch = os.getenv('CI_DEFAULT_BRANCH')
+    branch = os.getenv('CI_COMMIT_REF_NAME')
```

Comment:
Small nit: this variable should be named `git_ref`, since it can contain values that are not branch names but also tag names.